cli/units_cmd.go

```diff
@@ -38,6 +40,12 @@ func scanUnitsIntoConfig(cfg *config.Repository, quiet bool) error {
 		if err != nil {
 			return err
 		}
+
+		// Make sure that the file pointed to by cmdName is a regular executable file.
+		if err := checkRegularExecutableFile(cmdName); err != nil {
+			return fmt.Errorf("%s should be a regular executable file, but it's %s", cmdName, err)
```

Comment:
Probably better to move the entire error into that func, so this can be just:

```
if err := checkRegularExecutableFile(cmdName); err != nil {
    return err
}
```